docs/source/cli.rst

```diff
@@ -69,4 +69,4 @@ As with all other commands, you can also pass ``--help`` to both ``join`` and ``
 ``kasa --help``
 ***************
 
-.. program-output:: kasa --help
+.. program-output:: kasa --help --debug
```

Comment:
Any specific reason for adding this?

Reply:
For some reason running `kasa --help` returns an error code of 0, something to do with async click.  The echo saying `Run with --debug` causes the docs compilation to raise an error for some reason.

Reply:
Ah, that's a bug in our cli exception handler, which should be fixed. It currently captures the click internal exit value exception, so you will see this error also when running the cli locally.

Reply:
Yeah I wasn't sure the right way to fix it because the `Exit` exception is internal.

Reply:
I think we could just capture and handle it on our own, see #935

Reply:
That fixed it